openmdao_book/features/core_features/working_with_groups/add_subsystem.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>Missing example for <code>Promote the input and output of Components from subgroup level up to top level</code>  right before this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Added and pushed up to my <code>groups</code>  branch</p>